crates/rpc/rpc/src/eth/api/call.rs

```diff
@@ -222,11 +227,14 @@ where
         // transaction requires to succeed
         let gas_used = res.result.gas_used();
         // the lowest value is capped by the gas it takes for a transfer
-        let mut lowest_gas_limit = MIN_TRANSACTION_GAS;
+        let mut lowest_gas_limit =
```

Comment:
please keep the changes related to the PR,
it makes it harder to review if there are additional changes like this

Reply:
Sorry, but these changes were needed for `test_basic_eth_calls` to pass. Should I move them to a separate PR?

This one in particular resulted in a _"intrinsic gas too high"_ error message because the gas limit passed to _revm_ was lower than the base cost of the transaction (which by default is a `TransactTo::Create`).